guacamole-docker/bin/start.sh

```diff
@@ -1164,6 +1174,19 @@ if [ -n "$API_SESSION_TIMEOUT" ]; then
     associate_apisessiontimeout
 fi
 
+# Maximum number of bytes to accept within the entity body of any particular HTTP request
+set_optional_property "api-max-request-size" "$API_MAX_REQUEST_SIZE"
+
+# A comma-separated whitelist of language keys to allow as display language 
```

Comment:
I would suggest changing this to:
`A comma-separated list of language keys to allow...`

Reply:
ok